packages/cozy-konnector-libs/src/libs/saveBankingDocuments.js

```diff
@@ -1,36 +1,40 @@
 /**
- * Combines the features of `saveFiles`, `filterData`, `addData` and `linkBankOperations` for a
- * common case: bills.
- * Will create `io.cozy.bills` objects. The default deduplication keys are `['date', 'amount', 'vendor']`.
- * You need the full permission on `io.cozy.bills`, full permission on `io.cozy.files` and also
- * full permission on `io.cozy.bank.operations` in your manifest, to be able to * use this function.
+ * Combines the features of `saveFiles`, `filterData`, `addData` and `linkBankOperations` for any
+ * doctype.
+ * Will create `io.cozy.bills` documents by default or any specified doctype.
+ * The default deduplication keys are `['date', 'amount', 'vendor']`.
+ * You need the full permission on the doctype, full permission on `io.cozy.files` and also
+ * full permission on `io.cozy.bank.operations` in your manifest, to be able to use this function.
  *
  * Parameters:
  *
  * - `documents` is an array of objects with any attributes :
  * - `fields` (object) this is the first parameter given to BaseKonnector's constructor
  * - `options` is passed directly to `saveFiles`, `hydrateAndFilter`, `addData` and `linkBankOperations`.
+ *      - `doctype` option has `io.cozy.bills`
+ *      - `noBanking` option deactivates linkBankOperations. Default value is false.
```

Comment:
Since we may want to prevent linking with bank operations, doesn't this mean there will be other possible uses of these money-related documents? Then what about `saveFinancialDocuments()` as a name (with *financial* as in *money-related*)?